### ISSUE TITLE
Spacing prop does not accept list value

### ISSUE THREAD

**Issue by Davydhh** [reflex-dev/reflex#1919]
Hi guys, trying to make my app responsive I've set the 'spacing' prop of the hstack with a list, in order to support all screen sizes, but I got the error: TypeError: Invalid var passed for prop spacing, expected type <class 'str'>, got value [5, 10, 15, 20, 25] of type <class 'list'>.

Thank you

**Comment by Lendemor** [comment]
`spacing` is a prop that come from Chakra.

I don't see anything in their docs on the stack page that suggest it would accept a list instead of a str value.

**Comment by Davydhh** [comment]
Yes, I know, but a team member advised me to make an issue about this.

**Comment by Davydhh** [comment]
Yes, I know, but a team member advised me to make an issue about this.
